Services/ActiveRecord/Connector/Select/class.arSelectCollection.php

```diff
@@ -16,13 +16,12 @@ class arSelectCollection extends arStatementCollection {
 	public function asSQLStatement() {
 		$return = 'SELECT ';
 		if ($this->hasStatements()) {
-			foreach ($this->getSelects() as $select) {
-				$return .= $select->asSQLStatement($this->getAr());
-				if ($select != end($this->getSelects())) {
-					$return .= ', ';
-				}
-			}
-		}
+       $activeRecord = $this->getAr();
```

Comment:
There seem to be Spaces instead of Tabs for indention, could you change this? Rest seems absolutely fine, Thanks!

Reply:
Ah, that explains why indentation looked strange on github, I'm using spaces by default. :smiley: 

**fixed**